pkgs/applications/version-management/git-and-tools/git-dit/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, fetchFromGitHub, pandoc }:
+
+stdenv.mkDerivation rec {
+  name = "git-dit-${version}";
+  version = "0.1.0";
+
+  buildInputs = [ pandoc ];
+
+  src = fetchFromGitHub {
+    owner = "neithernut";
+    repo = "git-dit";
+    rev = "v${version}";
+    sha256 = "1rvp2dhnb8yqrracvfpvf8z1vz4fs0rii18hhrskr6n1sfd7x9kd";
+  };
+
+  # the Makefile doesn’t work, we emulate it below
+  dontBuild = true;
+
+  postPatch = ''
+    # resolve binaries to the right path
+    sed -e "s|exec git-dit-|exec $out/bin/git-dit-|" -i git-dit
+
+    # we change every git dit command to the local subcommand path
+    # (git dit foo -> /nix/store/…-git-dit/bin/git-dit-foo)
+    for script in git-dit-*; do
+      sed -e "s|git dit |$out/bin/git-dit-|g" -i "$script"
```

Comment:
Yeah, that’s a bug in the release.
It’s `for script in git-dit-*`, so `git-dit` isn’t in that loop.